client/src/components/Telepath/Telepath.test.js

```diff
@@ -0,0 +1,158 @@
+/* eslint-disable dot-notation */
+
+import Telepath from './Telepath';
+
+const telepath = new Telepath();
+
+class Artist {
+  constructor(name) {
+    this.name = name;
+  }
+}
+
+telepath.register('music.Artist', Artist);
+
+class Album {
+  constructor(title, artists) {
+    this.title = title;
+    this.artists = artists;
+  }
+}
+
+telepath.register('music.Album', Album);
+
+describe('Telepath', () => {
+  it('can unpack objects', () => {
+    const beyonce = telepath.unpack({ _type: 'music.Artist', _args: ['Beyoncé'] });
+    expect(beyonce.name).toBe('Beyoncé');
```

Comment:
Probably should have a `expect(beyonce).toBeInstanceOf(Artist)` here?

Reply:
Good call. This was my first time using jest, so I wasn't being very adventurous :-)